docs/2_registration_project_access_session.md

```diff
@@ -0,0 +1,162 @@
+~~~
+ls -R data/
+~~~
+
+> example_directory_1     example_directory_2
+> 
+> data/example_directory_1: sub_directory_1
+> 
+> data/example_directory_1/sub_directory_1: example_file_1.txt
+> 
+> data/example_directory_2: example_file_2.txt      example_file_5.txt  
+> sub_directory_2
+> 
+> data/example_directory_2/sub_directory_2: example_file_3.txt     
+> example_file_4.txt
+
+On windows, you can use the `dir` command or manually explore the structure through the file explored.
+
+
+The general upload command is [`dds data put`.](https://scilifelabdatacentre.github.io/dds_cli/data/#dds-data-put)
+~~~
+dds data put --project "<Project ID>" --source "<File or directory to upload>"
+~~~
+
+> There are a number of other optional flags, which cab be found on the documentation link just above.
+
+> NOTE: As of today, DDS can break when there are heavy uploads. We are working on fixing it, but on the meantime, we recommned to ZIP files when the uploads are big (Hundreds of files).
+
+
+ - [ ] TASK: Upload the files in the data folder. Remember to change the **project ID** and the **source** from the command above
+
+If you have forgotten the project ID, you can list all active projects you have access with: [Docs](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-ls)
+
+~~~
+dds project ls
+~~~
+
+# List contents
+
+You can interactively list the contents of a project with the [`ls` command:](https://scilifelabdatacentre.github.io/dds_cli/data/#dds-data-ls)
+~~~
+dds data ls --project "<Project ID>"
+~~~
+
+However, you can view all files and directories in a project as a tree structure, using the `--tree` option.
+
+~~~
+dds data ls --project "<Project ID>" --tree
+~~~
+
+![Screenshot of the files structure in the project](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-data-ls-tree.svg)
+
+ - [ ] TASK: Verify that all the data has being uploaded succesfully.
+
+# Release project and download data
+
+> Currently DDS does not support to resume downloads, if your download is interrupted you will need to restart it.
+
+1. In order for researchers to get access to the data, you need to [change the status](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-access) to `Available`. If a researcher tries to download data and the project is stil in progress they will get an error.
+
+~~~
+dds project status release --project "<Project ID>"
+~~~
+
+![enter image description here](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-project-status-release.svg)
+##
+
+After releasing the project, any curent researcher associated with the project will recieve a notification by email.
+
+2. To [invite researchers](dds%20user%20add%20%5BEmail%20address%5D%20--role "&lt;Account role&gt;quot; --project quot;&lt;Project ID&gt;") that will download the data to the project:
+
+~~~
+dds project access grant --project "<Project ID>" --email "<email>"
+~~~
+
+or
```

Comment:
```suggestion
for giving access to an already existing account, or
```

Reply:
@valyo - given this suggestion, would you disagree with the text (see discussion immediately above this one) that says the commands are the same?